apps/browser/src/autofill/utils/index.ts

```diff
@@ -360,3 +360,42 @@ export function throttle(callback: (_args: any) => any, limit: number) {
     }
   };
 }
+
+/**
+ * Gathers and normalizes keywords from a potential submit button element. Used
+ * to verify if the element submits a login or change password form.
+ *
+ * @param element - The element to gather keywords from.
+ */
+export function getSubmitButtonKeywordsSet(element: HTMLElement): Set<string> {
+  const keywords = [
+    element.textContent,
+    element.getAttribute("type"),
+    element.getAttribute("value"),
+    element.getAttribute("aria-label"),
+    element.getAttribute("aria-labelledby"),
+    element.getAttribute("aria-describedby"),
+    element.getAttribute("title"),
+    element.getAttribute("id"),
+    element.getAttribute("name"),
+    element.getAttribute("class"),
+  ];
+
+  const keywordsSet = new Set<string>();
+  for (let i = 0; i < keywords.length; i++) {
+    if (typeof keywords[i] === "string") {
+      keywords[i]
+        .toLowerCase()
+        .replace(/[-\s]/g, "")
+        .replace(/[^a-zA-Z0-9]+/g, "|")
+        .split("|")
```

Comment:
Same feedback from 
https://github.com/bitwarden/clients/pull/10617#discussion_r1733302377 and
https://github.com/bitwarden/clients/pull/10617#discussion_r1733306444

```suggestion
        .split(/[^\p{L}\|]+/gu, "|")
```

Reply:
Great suggestion, I've applied this regex.

Reply:
^ non-blocking, per offline discussion